umh-core/docs/production/sizing-guide.md

```
@@ -12,14 +12,18 @@
 
 ## Recommended Starting Point
 
-**Start with → 2 vCPU · 4 GB RAM · 40 GB SSD**
+**Start with → 4 vCPU · 4 GB RAM · 40 GB SSD**
 
 #### What that box handles
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)